tests/toplevel_unit.c

```diff
@@ -120,6 +120,41 @@ void test_example() {
     roaring_bitmap_free(r3);
 }
 
+bool check_bitmap_from_range(uint32_t min, uint32_t max, uint32_t step) {
+    roaring_bitmap_t *result = roaring_bitmap_from_range(min, max, step);
+    assert_non_null(result);
+    roaring_bitmap_t *expected = roaring_bitmap_create();
+    assert_non_null(expected);
+    for(uint32_t value = min ; value < max ; value += step) {
+        roaring_bitmap_add(expected, value);
+    }
+    bool is_equal = roaring_bitmap_equals(expected, result);
```

Comment:
@fsaintjacques 

Here I would think that printing out what is failing exactly can speed up debugging considerably. 

And where is the harm in @Ezibenroc's code?